resources/allowed-github-topics.properties

```
@@ -60,10 +57,6 @@ page-decorator
 parameter
 performance
 pipeline
-pipeline-graph-analysis
-plugin-external
-plugin-misc
-plugin-post-build
 plugin-test
```

```suggestion
plugin-test
```

Only one plugin has this label AFAICT, should just be `test`?

---

Good 👀 , fixed in last commit.